src/qt/rpcconsole.cpp

```diff
@@ -616,8 +616,16 @@ void RPCConsole::on_lineEdit_returnPressed()
 
     if(!cmd.isEmpty())
     {
-        message(CMD_REQUEST, cmd);
         Q_EMIT cmdRequest(cmd);
+        // Remove passphrases from history
+        QStringList cmdList = cmd.split(' ');
```

Comment:
That doesn't matter since only the command (first in the list) and the timeout (last in the list) are actually needed. Anything else in between doesn't matter.